packages/react-scripts/template/src/Toast.css

```diff
@@ -0,0 +1,39 @@
+.Toast {
+	position: fixed;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	padding: 16px;
+	margin: auto;
+	transition: 250ms;
+	background: #222;
+	color: white;
+	line-height: 24px;
+	border-top-left-radius: 2px;
+	border-top-right-radius: 2px;
+	display: flex;
+}
+
+.Toast.active {
+	transform: translateY(0);
+}
+
+.Toast.inactive {
+	transform: translateY(100%);
+}
+
+.Toast a,
+.Toast-button {
```

Comment:
Can we make this behave more like a link?

`cursor: pointer`

`Toast-button:hover { text-decoration: underline }`